tests/test_framework/utils.py

```diff
@@ -17,18 +18,18 @@
 
 from ephemeral_port_reserve import reserve
 from test_framework import serializations
-from typing import Optional
+from typing import Optional, Tuple
 
 
-TIMEOUT = int(os.getenv("TIMEOUT", 60))
-TEST_DEBUG = os.getenv("TEST_DEBUG", "0") == "1"
+TIMEOUT = int(os.getenv("TIMEOUT", 100))
+TEST_DEBUG = os.getenv("TEST_DEBUG", "1") == "1"
```

Comment:
Don't change the default value: if you want debug log traces, set the `TEST_DEBUG` env variable to `1` when running the tests.